config/helm/deployment-args.yaml

```diff
@@ -14,4 +14,6 @@ spec:
             - --metrics-bind-addr=0.0.0.0:8080
             - --feature-gates=EKS=true,EKSEnableIAM=true,EKSAllowAddRoles=false,EKSFargate=false,MachinePool=true,EventBridgeInstanceState=false,AutoControllerIdentityCreator=true,ExternalResourceGC=true,BootstrapFormatIgnition=true
             - --watch-filter=capi
+            # We need to set the sync-period to 2m to improve scale-up time, currently there is no watch in the capa controller that update the provideridlist
```

Comment:
If we have an issue to track this problem, I'd be good to link it. Otherwise, this is enough.

Reply:
added links